modules/CreateTasks.js

```diff
@@ -0,0 +1,73 @@
+import { GetTasks, GetLastTask } from './ReadTasks.js';
+import StoreTask from './UpdateTasks.js';
+import EditTask from './EditTasks.js';
+import { SelectTask } from './RemoveTask.js';
+
+const mainUl = document.getElementById('main-tasks');
+
+export const AddTask = (description) => {
+  const lastTask = GetLastTask();
+  const getLocalData = GetTasks();
+  const newTask = {
+    id: lastTask.id,
+    description,
+    completed: false,
+    index: lastTask.index,
+  };
+  getLocalData.push(newTask);
+  StoreTask(getLocalData);
+  return newTask;
+};
+
+const DisplayTask = (task) => {
+  const {
+    id, description, completed,
+  } = task;
+
+  const mainLi = document.createElement('li');
+  mainLi.classList.add(
+    'list-group-item',
+    'd-flex',
+    'justify-content-between',
+  );
+  mainLi.id = id;
+
+  const checkPContainer = document.createElement('div');
```

Comment:
Please initialize **all** variables at the top or bottom of your file for best practices, since it makes it easier for fellow peers to expect where to look for it when reviewing your code.

Reply:
Thanks for your comments but I think I can not create it at the top of the file because it's creating according to a function, so it need to be inside the `DisplayTask` function